Modules/posixmodule.c

```diff
@@ -8041,7 +8041,7 @@ os_dup2_impl(PyObject *module, int fd, int fd2, int inheritable)
     res = fd2; // msvcrt dup2 returns 0 on success.
 
```

Comment:
Please document somewhere the rationale for "fd != fd2" tests with a reference to "bpo-32862".